### ISSUE TITLE
The time needed for finetuning a model is inconsistant with the reported result

### ISSUE THREAD

**Issue by yzxyzh** [artidoro/qlora#77]
Hi!

Thanks for your wonderful work! currently we are trying to reproduce your result using a single A100 80G graphics card. We tried to train the llama-30b with your default settings, i.e., batchsize = 1, gradient accumulation=16, dataset is alpaca, but the time needed for training is estimated to be 80-90hours, which differs significantly with the result reported by you, which is about 24 hours for a 65b model. I'm wondering what could be the reason?

Thanks.

**Comment by yzxyzh** [comment]
Later we found that the author uses a training set with only about ~9000 examples, not the alpaca. This matches our observation.

**Comment by KKcorps** [comment]
@yzxyzh do you notice weird results from the model once trained if not loaded in 4-bit mode?
